game-app/game-core/src/main/java/games/strategy/engine/data/GamePlayer.java

```diff
@@ -24,7 +25,8 @@ public class GamePlayer extends NamedAttachable implements NamedUnitHolder {
   private static final String DEFAULT_TYPE_AI = "AI";
   private static final String DEFAULT_TYPE_DOES_NOTHING = "DoesNothing";
 
-  public static final GamePlayer NULL_PLAYERID =
+  @RemoveOnNextMajorRelease
+  private static final GamePlayer NULL_PLAYERID =
```

Comment:
What speaks against straight up removing this field? Static fields don't impact serialization, right?

Reply:
Actually, looking more, I think the reason I kept it is that references to this object will exist in the game and since it's a custom subclass, we'll get a class not found error if we delete, I think?

Reply:
No Idea how deserialization of anonymous classes works. But is the null player ever being serialized, or always just used for comparison?

Reply:
It does get serialized, since it's the default owner for territories and units there when an explicit player isn't listed in the map XML.